### PR TITLE
Homepage stats update

### PR DESCRIPTION
**Pull request by joaquimadraz**
### TODO
- [x] Stats order are different on smaller screen than on medium/larger screens
- [x] Refactor CSS

### Motivation

- [Design](https://zpl.io/25g9r9W)
- [Related story](https://trello.com/c/9YuAu2m6/129-new-stats-for-home-page)

### CODE REVIEW

**Comment by joaquimadraz** [comment]
[New stats for Home page](https://trello.com/c/9YuAu2m6/129-new-stats-for-home-page)

File: site/components/case-study-overview/index.js [hunk 1]
```diff
@@ -32,74 +37,100 @@ const CaseStudyOverview = () => (
...
         </a>
       </div>
       <div className={styles.caseCompanies}>
-        <img alt="The logo of Selfridges" src={selfridgesPNG} />
-        <img alt="The logo of Tesco" src={tescoPNG} />
-        <img alt="The logo of the BBC" src={bbcPNG} />
-        <img alt="The logo of BMW" src={bmwPNG} />
-        <img alt="The logo of HSBC" src={hsbcPNG} />
-        <img alt="The logo of Car Trawler" src={cartrawlerPNG} />
+        <div className={styles.caseCompaniesLogo}>
```
Comment (matt-paul) [inline site/components/case-study-overview/index.js]: Should this be a component now to reduce duplication?

File: site/components/case-study-overview/style.css [hunk 2]
```diff
@@ -107,66 +158,134 @@
...
+    margin: 0 auto;
+    max-width: 1160px;
+  }
+
+  .caseCompaniesLogo {
+    width: auto;
+    margin: 10px auto;
+  }
+
+  .caseCompaniesLogo img {
```
Comment (matt-paul) [inline site/components/case-study-overview/style.css]: Is this allowed? 😀🤔

File: site/components/case-study-overview/index.js [hunk 3]
```diff
@@ -1,29 +1,27 @@
...
+import CaseStudyCompanyLogos from '../case-study-company-logos';
 
-/* SVG figure imports */
-import ftFigureSVG from './SVG/ft-figure.svg';
-import fortnumFigureSVG from './SVG/fortnum-figure.svg';
-import camdenFigureSVG from './SVG/camden-figure.svg';
-import skyFigureSVG from './SVG/sky-figure.svg';
-import arrowSVG from '../../../assets/images/SVG/arrow.svg';
+const daysSinceFortumStarted = () => {
+  const startDate = moment([2014, 5, 14]);
```
Comment (chris-brookes) [inline site/components/case-study-overview/index.js]: Start date should be [2014, 4, 14]. The month count starts at 0, so May will be 4.

**Comment by chris-brookes** [comment]
@dpiatek - I spoke to Joaquim and they made a number of design changes due to page widths, but didn't update the Zeplin, so that's why a load of the fonts etc. are different to the Zeplin design. It may be good for whoever tests this to chat to him before they do it.

**Pull request merged**